src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const [value, key] of Object.entries(robot)) {
+    if (fixedRobot.hasOwnProperty([key])) {
```

Comment:
Why do you pass your string in an array, it's redundant
```suggestion
    if (fixedRobot.hasOwnProperty(key)) {
```